examples/api/test/widgets/shortcuts/shortcuts.0_test.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/widgets/shortcuts/shortcuts.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('ShortcutsExampleApp', () {
```

Comment:
Single` group` not necessary.

Reply:
I used a group here to get the following test output, while not copy-pasting `ShortcutsExampleApp` into each test name:
```
✓ ShortcutsExampleApp displays correct labels
✓ ShortcutsExampleApp updates counter on arrowUp and arrowDown press
```

While without group the output will be the following:
```
✓ displays correct labels
✓ updates counter on arrowUp and arrowDown press
```

Is this fine for you?

Reply:
I can understand the benefits of this. However, existing tests have more descriptive title and do not repeat the example name as it is already indicated example test file naming scheme.

E.g. This describe the example bevhaior better without group.
https://github.com/flutter/flutter/blob/774ef8acf4ce4d8de739371e279c4f8296edccb4/examples/api/test/material/button_style_button/button_style_button.icon_alignment.0_test.dart#L5-L10

Reply:
Instead of example title, it describes the actual property used to update the icons